dev/com.ibm.ws.jpa_spec21_fat/test-applications/TxSynchronization/src/com/ibm/ws/jpa/commonentities/jpa10/simple/SimpleVersionedEntity10.java

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jpa.commonentities.jpa10.simple;
+
+import javax.persistence.Basic;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+@Entity
+@Table(name = "CMN10_SimpleVEnt")
+public class SimpleVersionedEntity10 implements ISimpleVersionedEntity10 {
+    @Id
+    private int id;
+
+    private String strData;
+    private char charData;
```

Comment:
True, I haven't tested it against anything other than Derby.  But this is one of those FATs where testing against more than Derby is arguably ineffectual, since it's verifying that the JPA Runtime integration is doing the right thing.